java-bootcamp-bahia/src/exc03/Room.java

```diff
@@ -0,0 +1,7 @@
+package exc03;
+
+public interface Room {

```

Comment:
You should include in this interface all the methods implemented on EmergencyRoom and CommonRoom classes to avoid the need to use cast in the BuildRooms method in each of the concrete builders.